tests/python/relay/test_dataflow_pattern.py

```diff
@@ -1793,3 +1825,4 @@ def test_partition_constant_embedding():
     test_IfPattern()
     test_match_if()
     test_no_match_if()
+    test_rewrite_once()
```

Comment:
As we are making unit tests more manageable, could you help change to the following?
```
if __name__ == "__main__":
    pytest.main([__file__])
```